src/main/scala/enum/NonStellarLibrarySpectrum.scala

```diff
@@ -47,5 +49,19 @@ object NonStellarLibrarySpectrum {
   case object Uranus            extends NonStellarLibrarySpectrum("Uranus", "Uranus", "Uranus", "Uranus")
   case object Neptune           extends NonStellarLibrarySpectrum("Neptune", "Neptune", "Neptune", "Neptune")
 
+  val all = List(
+    EllipticalGalaxy, SpiralGalaxy, QS0, QS02, OrionNebula, PlanetaryNebula, PlanetaryNebula2,
+    PlanetaryNebula3, StarburstGalaxy, PmsStar, GalacticCenter, Afgl230, Afgl3068, AlphaBoo,
+    AlphaCar, BetaAnd, BetaGru, GammaCas, GammaDra, L1511Irs, NGC1068, NGC2023, NGC2440, OCet,
+    OrionBar, Rscl, Txpsc, Wr104, Wr34, Mars, Jupiter, Saturn, Uranus, Neptune
+  )
+
+def fromTag(tag: String): Option[NonStellarLibrarySpectrum] =
+  all.find(_.tag === tag)
+
+def fromTagIgnoreCase(tag: String): Option[NonStellarLibrarySpectrum] =
```

Comment:
By convention (it seems) GraphQL enumerations are uppercased, so to look up enum elements I need a case-insensitive lookup method.